### PR TITLE
Selection page

### PR DESCRIPTION
**Pull request by ngdo-pro**
Remember to gulp and doctrine:fixture:load

### CODE REVIEW

File: app/Resources/translations/baptism_selection.fr.xlf~ [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <source>search_result.title</source>
+        <target>résultat de la recherche</target>
+      </trans-unit>
+      <trans-unit id="7c94aa4a257438c51bd08d0cc6a412e4" resname="back_to_search">
+        <source>back_to_search</source>
+        <target>retour à la recherche</target>
+      </trans-unit>
+      <trans-unit id="942c37ac9b657b3dfc7f0bf098fe68fe" resname="no_baptism_found">
+        <source>no_baptism_found</source>
+        <target>aucun baptême correspondant à vos critères a été trouvé</target>
```
Comment (wafaworks) [inline app/Resources/translations/baptism_selection.fr.xlf~:22]: just a syntax error : "a été trouvé" must be replaced by "n'a été trouvé"

File: web/dev/sass/modules/_typography.sass [hunk 2]
```diff
@@ -7,7 +7,6 @@ $btn-font: 'Droid Sans', sans-serif
 $main-title-font: 'Asap', sans-serif
 $title-font: 'Droid Sans', sans-serif
 $title-font2: 'Asap', sans-serif
-$main-title-font: 'Asap', sans-serif
```
Comment (wafaworks) [inline web/dev/sass/modules/_typography.sass:10]: did you supress this ? be careful because we're using this in many sass files..

**Review by wafaworks — CHANGES_REQUESTED** [review]
only a syntax error found and a question about a sass variable. Good job though! 👍

File: app/Resources/translations/baptism_selection.fr.xlf~ (hunk 1 above)
Comment (ngdo-pro) [reply app/Resources/translations/baptism_selection.fr.xlf~:22]: "aucun" already induce negation, "n'a" could be added, but is not mandatory, i'll add it if you want

File: web/dev/sass/modules/_typography.sass (hunk 2 above)
Comment (ngdo-pro) [reply web/dev/sass/modules/_typography.sass:10]: please, pay attention, line 7 and line 10 were the same, we don't need duplicate content.

**Review by Kaji-U — APPROVED** [review]
Ok for me

**Review by shog189 — APPROVED** [review]
:+1:

**Review by wafaworks — APPROVED** [review]

**Pull request merged**